pkg/nodeconfig/nodeconfig.go

```diff
@@ -12,6 +12,7 @@ import (
 	ignCfgTypes "github.com/coreos/ignition/v2/config/v3_4/types"
 	"github.com/go-logr/logr"
 	configv1 "github.com/openshift/api/config/v1"
+	mcfg "github.com/openshift/api/machineconfiguration/v1"
```

Comment:
nit: maybe mcfg could be changed to something more descriptive

Reply:
I think is fine, that is the existing pattern https://github.com/openshift/windows-machine-config-operator/blob/master/pkg/ignition/ignition.go#L11